2-collectors/scala-stream-collector/examples/config.hocon.sample

```diff
@@ -180,6 +180,16 @@ collector {
       timeLimit = {{bufferTimeThreshold}}
     }
   }
+
+  # Configuration of prometheus http metrics
+  metrics {
+    # If metrics are enabled then all requests will be logged as prometheus metics
+    # and '/metrics' endpoint will return the report about the requests
+    enabled = false
+    # Custom buckets for http_request_duration_seconds_bucket duration metric
+    # durationBuckets = [0.1, 3, 10]
```

Comment:
this needs to be uncommented otherwise configuration parsing won't work :+1:

Reply:
This is commented on purpose. 
By default built-in buckets are used. If they does not meet the needs of application then they can be configured via hocon.
The parameter `durationBuckets` is specified as optional and is allowed to be omitted